From e47fc8db812491fcbf53b30a61f13841c01f6a35 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 30 Jun 2023 10:22:50 +0200
Subject: [PATCH 0157/1695] clk: stm32mp13: remove the iwdg1 from the critical
 clocks list

The iwdg1 APB clock was set as critical because needed for
propagating the early interrupt to the GIC.

With the stm32_iwdg driver taking care of it, the clock doesn't
need to be critical and can be removed from the list.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e0c69708aaf18d0edecf4abb8c6a7dd1bc73fc5b clk: stm32mp13: remove the iwdg1 from the critical clocks list]
Change-Id: I8988c103a642c4290d4ce5c54d69276e4421d129
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
---
 core/drivers/clk/clk-stm32mp13.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/drivers/clk/clk-stm32mp13.c b/core/drivers/clk/clk-stm32mp13.c
index cd46808a8..ca93f7567 100644
--- a/core/drivers/clk/clk-stm32mp13.c
+++ b/core/drivers/clk/clk-stm32mp13.c
@@ -2580,7 +2580,6 @@ static bool clk_stm32_clock_is_critical(struct clk *clk __maybe_unused)
 		&ck_rtcapb,
 		&ck_tzc,
 		&ck_etzpcb,
-		&ck_iwdg1apb,
 		&ck_bsec,
 		&ck_stgen_k,
 		&ck_bkpsram,
-- 
2.47.2

